src/incflo.H

```diff
@@ -370,6 +370,8 @@ private:
     //////////////////////////////////////////////////////////////////////////////////////////////
 
     void ComputeDrag();
+    
+    double ComputeKineticEnergy();
```

Comment:
Because of a CUDA limitation on extended lambda, this function needs to be `public`.

Reply:
I've learned a lot with this pull-request, @WeiqunZhang thank you for taking the time to show me the ropes.

We need to get our CUDA builds running on Eagle. Have you successfully built with CUDA? I can't get CUDA or OMP to build.

Reply:
Yes, I can get it to build on my desktop.  But I haven't tried to run it yet.  I am currently going through the code and fixing a couple of issues in my own fork.  I will merge my fork when it's ready.

Reply:
If you need help on Eagle, let me know.  Even though I don't have access to Eagle, I might have some suggestions.